components/cnc-tech/3020-xx-0300-00.stanza

```diff
@@ -0,0 +1,103 @@
+#use-added-syntax(jitx)
+defpackage ocdb/cnc-tech/_3020-xx-0300-00:
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+  import ocdb/design-vars
+  import ocdb/land-patterns
+  import ocdb/tolerance
+  import ocdb/box-symbol
+  import ocdb/rules
+  import ocdb/generic-components
+  import ocdb/symbols
+  import ocdb/land-protrusions
+
+pcb-landpattern lp-3020-xx-0300-00 (number-of-leads:Int, a-dim:Double):
+  make-n-pin-soic-landpattern(number-of-leads, 2.54, tol(9.5, 0.15), tol(8.75, 0.2), tol(a-dim, 0.35), tol((9.5 - 2.54) / 2.0, 0.15), tol(0.64, 0.01), "row-major" )
```

Comment:
You can do something like this which is a bit closer to named arguments and more self-documenting 
```
make-n-pin-soic-landpatetern(number-of-leads, pitch, lead-span, package-length, terminal-length, terminal-width, "row-major") where : 
  val pitch = 2.54
  val lead-span = tol(9.5, 0.15)
  val package-length = tol(8.75, 0.2)
  val package-width = tol(a-dim, 0.35)
  val terminal-length = tol((9.5 - 2.54) / 2.0, 0.15),
  val terminal-width = tol(0.64, 0.01)
```

Reply:
fixed